docs/Community/Committers.md

```diff
@@ -0,0 +1,50 @@
+Project Committers
+==================
+
+Current Status
+--------------
+
+The following people have been granted commit permissions (that is, write
+access) to the Open Enclave SDK by the Community Governance Committee. The area
+column describes which technical areas each committer is most interested in, and
+therefore should usually be consulted for changes relating to that area.
+However, it is up to each committer to determine who should review which PR, and
+when to merge it. Remember that a PR must not be merged if a committer objects;
```

Comment:
Is putting "request changes" against a PR = objection? Are there other ways to object?

Reply:
@radhikaj , that was the original wording. I've attempted to clarify in the latest version. Is this better?